Gems/SurfaceData/Code/Include/SurfaceData/SurfaceDataTypes.h

```diff
@@ -188,7 +182,7 @@ namespace SurfaceData
         //! @return The pointer to the tag that's found, or end() if it wasn't found.
         const AzFramework::SurfaceData::SurfaceTagWeight* FindTag(AZ::Crc32 tag) const;
 
-        AZStd::fixed_vector<AzFramework::SurfaceData::SurfaceTagWeight, MaxSurfaceWeights> m_weights;
+        AZStd::fixed_vector<AzFramework::SurfaceData::SurfaceTagWeight, AzFramework::SurfaceData::Constants::MaxSurfaceWeights> m_weights;
```

Comment:
Suggestion: Could just make m_weights of type AzFramework::SurfaceTagWeightList now that the two are fully aligned.

Reply:
I'll make this change in a separate PR later